### PR TITLE
Fixed double call for CMC client

### PR DESCRIPTION
**Pull request by EnoRage**

### CODE REVIEW

File: market/rate/cmc/cmc.go [hunk 1]
```diff
@@ -17,14 +17,14 @@ type Cmc struct {
 	client *cmc.Client
 }
 
-func InitRate(api string, apiKey string, mapApi string, updateTime string) rate.RateProvider {
+func InitRate(mapApi string, updateTime string, client *cmc.Client) rate.RateProvider {
```
Comment (prazd) [inline market/rate/cmc/cmc.go]: InitRate(mapApi, updateTime string, client *cmc.Client)

File: market/ticker/cmc/cmc.go [hunk 2]
```diff
@@ -16,14 +16,14 @@ type Market struct {
 	client *cmc.Client
 }
 
-func InitMarket(api string, apiKey string, mapApi string, updateTime string) ticker.TickerProvider {
+func InitMarket(mapApi string, updateTime string, client *cmc.Client) ticker.TickerProvider {
```
Comment (prazd) [inline market/ticker/cmc/cmc.go]: InitMarket(mapApi, updateTime string, client *cmc.Client)

Comment (prazd) [reply market/ticker/cmc/cmc.go]: Or you can use struct for this

File: market/rate/cmc/cmc.go [hunk 3]
```diff
@@ -17,14 +17,14 @@ type Cmc struct {
 	client *cmc.Client
 }
 
-func InitRate(api string, apiKey string, mapApi string, updateTime string) rate.RateProvider {
+func InitRate(mapApi, updateTime string, client *cmc.Client) rate.RateProvider {
 	cmc := &Cmc{
```
Comment (vikmeup) [inline market/rate/cmc/cmc.go]: what is `mapApi` ?

File: config.yml [hunk 4]
```diff
@@ -23,6 +23,7 @@ market:
     widgetapi: https://widgets.coinmarketcap.com
     api_key: # YOUR_API_KEY
     map_url: https://raw.githubusercontent.com/trustwallet/assets/master/pricing/coinmarketcap
+    caching_duration: 1m
```
Comment (vikmeup) [inline config.yml]: `caching_duration` => `cache_duration`

File: market/rate/cmc/cmc.go [hunk 5]
```diff
@@ -17,14 +17,14 @@ type Cmc struct {
...
 		Rate: rate.Rate{
 			Id:         id,
 			UpdateTime: updateTime,
```
Comment (vikmeup) [inline market/rate/cmc/cmc.go]: `UpdateTime` => `UpdateFrequency`

**Pull request merged**